lib/ssl/derive.c

```diff
@@ -1,716 +0,0 @@
-/*
```

Comment:
How would you show it?  Just showing the filename and "deleted" means that you can't see what was deleted, which might be important.